packages/web/src/features/translation/TranslationView.tsx

```diff
@@ -68,19 +90,31 @@ function useTranslationQueries(language: string, verseId: string) {
         queryFn: ({ queryKey }) =>
           apiClient.verses.findVerseGlosses(queryKey[2], queryKey[1]),
       });
+      if (selectedLanguage) {
+        queryClient.ensureQueryData({
```

Comment:
```suggestion
        queryClient.prefetchQuery({
```

Should I do this instead, like #235?

Reply:
Yep

Reply:
Done